wikimedia/wikipedia-ios
# Diff design review feedback updates

**Phabricator:** https://phabricator.wikimedia.org/T344453

### Notes
* This PR adds some adjustments to the diff screen based on the design feedback described on the ticket

### Test Steps
1. Run the app and make sure the issues were addressed
2. If possible, test in different sizes of iPhones and iPads, rotate the screen 

###Screenshots 
<img src="https://github.com/wikimedia/wikipedia-ios/assets/31116510/f81be33c-2999-4997-93ec-ca31c73d5ff9" width="120">
